SamplePay/app/src/main/java/com/example/android/samplepay/model/PaymentDetails.kt

```diff
@@ -21,16 +21,20 @@ import org.json.JSONException
 import org.json.JSONObject
 
 data class PaymentAmount(
-    val currency: String,
-    val value: String
+    val currency: String, val value: String
 ) {
     companion object {
+        private val ISO_TO_SYMBOL_CURRENCY_MAP = mapOf(
```

Comment:
Minor nit: Java may have a currency formatter that could be used here.

See:
- https://docs.oracle.com/javase/8/docs/api/java/text/NumberFormat.html#getCurrencyInstance--

However, the only place invoking this app is the demo website, which is not going to use any other currency, I presume.

Reply:
Updated